src/engraving/libmscore/cmd.cpp

```diff
@@ -26,6 +26,7 @@
 */
 
 #include <assert.h>
+#include <algorithm>
```

Comment:
I just realised that you don't need `<algorithm>` either, since the `sort` you are using is a member of `std::list`. Sorry, I should have spotted that earlier...

Reply:
You're quite right!
I'll fix that.